### ISSUE TITLE
[Bug] Dead on arrival - too many errors

### ISSUE THREAD

**Issue by Mulli** [creativetimofficial/ct-now-ui-dashboard-pro-react#39]
### Version
v1.5.2
### Reproduction link
https://local.dev
### Operating System
Windows 10
### Device
Laptop i7-1065, 16GB RAM
### Browser & Version
Chrome126.0.6478.61
### Steps to reproduce
1. Unzip
2. npm install - completed with warnings
3. npm start - crash
### What is expected?
app running on localhost
### What is actually happening?
Error: digital ebvelope routines::unsupported

---
### Solution
Refund. Look for more updated dashboard product
### Additional comments
I wish I had a clue to the status on the packages prior to selecting the product

**Comment by throcha3** [comment]
I agree. Look at my console just for running the project. 26 errors. I am not even mentioning the css bugs...

![image](https://github.com/user-attachments/assets/4b3e1816-ae43-41ad-a772-28b53b2b419e)
